tinygrad/tensor.py

```diff
@@ -306,7 +306,10 @@ def min(self, axis=None, keepdim=False): return -((-self).max(axis=axis, keepdim
   def mean(self, axis=None, keepdim=False):
     out = self.sum(axis=axis, keepdim=keepdim)
     return out * (prod(out.shape)/prod(self.shape))
-
+  def std(self, axis=None, keepdim=False, unbiased=True):
+    n = self.shape[axis] if axis is not None else prod(self.shape)
```

Comment:
Not with this!

Reply:
But `prod(square_sum.shape)` can not pass test_std_axis with "not equal tolerance", however above can pass.

Reply:
This passes all the tests. Is it eligible? `(square_sum / ((prod(self.shape[:axis+1]) if axis is not None else prod(self.shape))-(1 if unbiased else 0))).sqrt()`